http-server-netty/src/main/java/io/micronaut/http/server/netty/NettyHttpServer.java

```diff
@@ -106,7 +109,7 @@
         value = ChannelOption.class,
         accessType = {TypeHint.AccessType.ALL_DECLARED_CONSTRUCTORS, TypeHint.AccessType.ALL_DECLARED_FIELDS}
 )
-public class NettyHttpServer implements NettyEmbeddedServer {
+public class NettyHttpServer implements NettyEmbeddedServer, OrderedCracResource {
```

Comment:
I don't think directly implementing crac interfaces is wise here. There should be another bean that is activated only when CraC is present that starts/stops the server